pokerapp/utils/draw_poker_table.py

```diff
@@ -0,0 +1,60 @@
+import io
+from typing import List
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from pokerapp.entity.player import Player
+
+oval_center = (0, 0)
+oval_a = 3
+oval_b = 2
+oval_angle = 0
+circle_radius = 0.5
+oval_offset = 0.2
+
+
+def __draw_oval(ax, center, a, b, angle, color='black', linewidth=1, linestyle='solid'):
+    t = np.linspace(0, 2 * np.pi, 100)
+    x = center[0] + a * np.cos(t) * np.cos(angle) - b * np.sin(t) * np.sin(angle)
+    y = center[1] + a * np.cos(t) * np.sin(angle) + b * np.sin(t) * np.cos(angle)
+    ax.plot(x, y, color=color, linewidth=linewidth, linestyle=linestyle)
+
+
+def __draw_circle(ax, center, radius, color='black', linewidth=1):
+    t = np.linspace(0, 2 * np.pi, 100)
+    x = center[0] + radius * np.cos(t)
+    y = center[1] + radius * np.sin(t)
+    ax.plot(x, y, color=color, linewidth=linewidth)
+
+
+def draw_poker_table(players: List[Player], current_player: Player):
```

Comment:
prolly in the end Game entity needs to be passed to draw out everything the table, and pokerbotview.py is anyway coupled with Game entity

Reply:
got it!